.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+'blank_issues_enabled': true
```

Comment:
I think this file is not needed. It contains link to reporter which does not exists for CLI.

AdGuardVPNCLI repo does not have this file.

Reply:
Or just link https://github.com/AdguardTeam/AdGuardCLI/issues :)